kiwi/bootloader/config/grub2.py

```diff
@@ -253,6 +254,22 @@ def setup_disk_image_config(
                 with open(config_file, 'w') as grub_config_file:
                     grub_config_file.write(grub_config)
 
+        if self.validate_use_of_linuxefi:
+            # On Systems that uses GRUB_USE_LINUXEFI we check if the
+            # grub2 config creation tool was able to detect that
+            # linuxefi/initrdefi commands have to be used. If not
+            # we fix this condition
+            with open(config_file) as grub_config_file:
+                grub_config = grub_config_file.read()
+                grub_config = re.sub(
+                    r'([ \t]+)linux', r'\1linuxefi', grub_config
```

Comment:
This would replace `linuxefi` with `linuxefiefi`.

Reply:
jaha my bad, needs fixing, thanks

Reply:
should be better now